src/pages/api/accounts/index.js

```diff
@@ -4,12 +4,13 @@ async function registerNewUser(req, res) {
       "Content-Type": "application/json",
       Authorization: `Token ${process.env.ACTNOW_API_KEY}`,
     });
-    const response = await fetch(`${process.env.ACTNOW_URL}/accounts/`, {
+    const response = await fetch(`${process.env.ACTNOW_URL}/v1/accounts/`, {
```

Comment:
Ola @kilemensi  I had updated vercel `ACTNOW_URL` to include the `v1` part(Forgot to mention it) but I've just removed it now.

Reply:
Cool, cool.